### DISCUSSION TITLE
Problema en comando .append kata 6

### DISCUSSION THREAD

**Discussion by CarlosCorales** [LaunchX-InnovaccionVirtual/CursoIntroPython#176] [Q&A] [answered]
alguien sabe por que no me funciona el comando .append?  anteriormente me funciono pero conforme avance puse un codigo mal, y cuando quise corregir los errores elimine varias lineas de codigos, hasta llegar a esa y no me funciona ahora el comando

![image](https://user-images.githubusercontent.com/99147223/153696235-c63c9002-44b7-46fb-b726-7d486031c1b4.png)

**Comment by CarlosCorales** [comment]
la captura no tiene el valor de lo que agregue, pero antes si puse pluton, pero no me funciono, esto la tome antes de poner el valor otra vez

**Comment by CarlosCorales** [comment]
![image](https://user-images.githubusercontent.com/99147223/153696260-b172c790-04a4-4501-a804-b67bb7d23de3.png)
 asi me aparece con el valor que quiero agregar

**Comment by OscarPedroza** [comment, marked as answer]
Buenas noches, la lista de planetas debe ir entre corchetes [] en lugar de parentesis.

**Reply by CarlosCorales** [reply]
ah okok, muchas gracias!

**Reply by JosueVerAlar** [reply]
Recuerda que las listas se crean con corchetes [] y las tuplas con paréntesis (). También es importante mencionar que las tuplas no se pueden modificar, las listas si. Saludos!
